### ISSUE TITLE
Show time since last block

### ISSUE THREAD

**Issue by dghaehre** [0xB10C/memo#18]
Would be nice to have a timer that displays the time since last block so we can get a feel for how close we are to getting the transaction confirmed.

**Comment by 0xB10C** [comment]
I'm a little ambivalent about this idea. 

On one hand the time since the last block doesn't imply the time till the next block (and not how close the confirmation of a specific tx is). For brevity I cite [the Bitcoin wiki article about confirmation times](https://en.bitcoin.it/wiki/Confirmation#Confirmation_Times):

> The block interval has an average of 10 minutes but not every block interval is exactly 10 minutes. It follows a statistical process known as a [poisson process](https://en.wikipedia.org/wiki/Poisson_point_process), where random events happen with the same probability in each time interval. Another way of expressing this is that the mining process has no memory, at every second a block has the same chance of being found. Poisson processes are well-understood but can be unintuative. 

On the other hand I like the feature from a UX perspective. Supplying a user waiting for the next block and his transaction to confirm with some information is good. 

 **What could be done:** 
1. Get the time of the last block 
2. Calculate the time difference as `diff` (e.g. last block 10 minutes ago)
3. Calculate `diff + 5min` as `diffPlusFive` (e.g. 15 minutes)
4. Calculate the block probability for a wait time of `diffPlusFive` (e.g. ~75% for 15 minutes)
5. Display "Chance of Block in the next 5 minutes: 75%" to the user.

Totally open for discussion.  

Resources: 
https://en.bitcoin.it/wiki/File:Block-cumulative-interval.png
https://gist.github.com/chris-belcher/647ccff5efc34511979767184153c3da

**Comment by timothyylim** [comment]
Perhaps we could do "Time since last block" for now and have an info icon that explains in more nuance the probability of the next block as according to the bitcoin wiki article mentioned above.

Although something like: "Chance of Block in the next 5 minutes: 75%" would be factually accurate, I fear that it will actually confuse non-technical users more than helping them.

It's a hard problem and I think we should tackle a proper estimation algorithm later:
https://github.com/0xB10C/memo/projects/1#card-19393560

**Comment by 0xB10C** [comment]
@dghaehre I think we've found a way to incorporate both our ideas in a new card on the site with a new chart. Since you opened this issue I though you might want to give feedback. This feature is still WIP, so everything is appreciated. One thing I found is that the chart looks quite dull color wise compared to the Current Mempool chart. But I'm not sure how to change that, without introducing colors without a meaning.

![image](https://user-images.githubusercontent.com/19157360/57073521-96812c00-6ce1-11e9-919b-1195f70caf8a.png)
